integration_tests/src/main/python/parquet_test.py

```diff
@@ -125,6 +125,19 @@ def test_ts_read_round_trip(spark_tmp_path, ts_write, ts_rebase, small_file_opt,
             conf={'spark.rapids.sql.format.parquet.smallFiles.enabled': small_file_opt,
                   'spark.sql.sources.useV1SourceList': v1_enabled_list})
 
+@pytest.mark.parametrize('ts_type', ['TIMESTAMP_MILLIS'])
+@pytest.mark.parametrize('ts_rebase', ['CORRECTED'])
+def test_parquet_write_ts_millis(spark_tmp_path, ts_type, ts_rebase):
```

Comment:
When Jason called out the issue with the comparison I noticed that this test is based off of a read test not a write test. It is comparing that we can write with the GPU something that both the CPU and GPU can read.

What we want is to verify that we can write something with both the CPU and the GPU and that both files can be read by the CPU to get the same answer.

see `test_write_round_trip` as an example of how we are doing that.

Reply:
Also might be good to move this test next to the other write tests...

Reply:
I have made the change. What confused me is that the outputTimestampType is only used while writing (rightly).